doc/documentation_data_format.md

```diff
@@ -436,7 +436,7 @@ provided) inside the measurement table.
 Expected to have the following columns in any (but preferably this)
 order:
 
-| plotId | [plotName] | plotTypeSimulation | plotTypeData | datasetId | ...
+| plotId | [plotName] | [plotTypeSimulation] | [plotTypeData] | [datasetId] | ...
```

Comment:
so far, I think datasetId and xValues are mandatory.
Question: If we want to change that, that would mean the only mandatory column would be plotId, does this make sense? What should then be plotted if just plotId is provided?

Reply:
Well, in a way it still makes sense, as it allows to specify values different from defaults only for a subset of optional columns. However, if there is only one column in vis_spec, maybe we should show some message.

Reply:
okay, sounds good :+1: